src/selectors/getAuthProvider.js

```diff
@@ -0,0 +1,15 @@
+import {createSelector} from 'reselect';
```

Comment:
This is to tell whether we are signed in with google or github
2 things here: 1 small and 1 big

1. I feel like this is messy thoughts on cleaning up

2. I think like there is some overlap here with isAuthenicated. Would it be better to have an 'isAuthenicatedGoogle` and `isAuthenicatedGithub` selectors?
Also In a future PR it might be easier to pass down the whole credential and render the hamburger items based on the auth provider indicated in the credential.

Reply:
Also to be extra cute you could create a higher-order selector e.g. `makeIsAuthenticatedWith` but might be a little much for just two actual cases.